apps/protailwind/src/pages/workshops/[module]/index.tsx

```diff
@@ -1,33 +1,44 @@
 import React from 'react'
 import {SanityDocument} from '@sanity/client'
-import {GetStaticPaths, GetStaticProps} from 'next'
+import {GetStaticPaths, GetStaticProps, GetServerSideProps} from 'next'
 import {getAllWorkshops, getWorkshop} from '../../../lib/workshops'
 import WorkshopTemplate from '../../../templates/workshop-template'
+import {getActiveProducts} from 'path-to-purchase-react/products.server'
+import {propsForCommerce} from '@skillrecordings/commerce-server'
+import {getToken} from 'next-auth/jwt'
+import {CommerceProps} from '@skillrecordings/commerce-server/dist/@types'
 
 export const USER_ID_QUERY_PARAM_KEY = 'learner'
 
-export const getStaticProps: GetStaticProps = async ({params}) => {
```

Comment:
we usually default to SSR on pages with commerce but I would still love to know how static approach would look like

Reply:
it could be loaded via trpc client side

Reply:
I used trpc so this is still static 🎁